src/components/token/TokenBalances.js

```diff
@@ -366,6 +370,51 @@ function TokenBalances({ maintenanceMode }) {
       {tokenId && renderTokensTable()}
     </div>
   );
+
+  const renderNewUi = () => (
+    <div className="container-title-page">
+      <p className="title-page">Token Balance</p>
+      {renderSearchField()}
+
+      <div className="token-balances-information-wrapper">
+        <h1>
+          {this.state.name === undefined
+            ? 'Hathor - HTR'
```

Comment:
`Hathor - HTR` should not be hardcoded.
We have a `state.serverInfo.native_token` which has a name and symbol that should be used here.

Reply:
How can I access the `serverInfo` object here? Since we're now in the refactored functional component, we don't have access to the previous state, and none of the defined state variables reference serverInfo.

Reply:
Nevermind, it's available through the redux store.

Reply:
Fixed as sugested. Please review when possible.